app/helpers/course_details_helper.rb

```diff
@@ -23,6 +23,12 @@ def route_title(route)
     t("views.forms.publish_course_details.route_titles.#{route}")
   end
 
+  def subjects_for_summary_view(subject, subject_two, subject_three)
+    additional_subjects = [subject_two, subject_three].reject(&:blank?).join(" and ")
+
+    [subject, additional_subjects].reject(&:blank?).join(" with ")
+  end
```

Comment:
Is this the proper way to display multiple subjects? `with` & `and` 


@gregknight1

Reply:
For two subjects: `a with b`

For three subjects: `a with b and c`

I've got another card to look at making them easier to read - probably copying some of the renaming things that publish does.

Reply:
something I did in the prototype but we've not tackled here is to recase the subjects appropriately so they're not title case. This needs a bit of work though as we wouldn't want to recase language names.